Fix bug in preprocessor directives that select the return type for main
--- ckwart.c.orig	2009-10-17 02:23:07
+++ ckwart.c	2026-01-22 13:37:14
@@ -513,6 +513,7 @@ int
 #else
 #ifdef STRATUS
 int
+#else
 #ifdef __GNUC__
 int
 #else
